docs/4.x/tests-github

```diff
@@ -0,0 +1,31 @@
+---
+category: Develop
+previous: tests-js-tracker
+next: tests-troubleshooting
+---
+# Running tests/builds on GitHub Action
+
+[Github Action](https://github.com/features/actions) is a continuous integration tool that will run tests/builds for a GitHub repository every time commits are pushed.
```

Comment:
There are actually a lot more triggers than only commits. We also use some like comments, cron, manual,...